src/passes/Precompute.cpp

```diff
@@ -390,6 +395,7 @@ struct Precompute
           for (auto* set : localGraph.getInfluences[get]) {
             work.push(set);
           }
+          worked = true;
```

Comment:
Can we have `optimizeLocals` return this bool rather than using separate state?

Reply:
Good idea, done.